server/graphql/item_variant/src/lib.rs

```diff
@@ -1,8 +1,43 @@
 use async_graphql::*;
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use service::auth::{Resource, ResourceAccessRequest};
 
 mod mutations;
 use self::mutations::*;
 
+#[derive(Default, Clone)]
+pub struct ItemVariantQueries;
+
+#[Object]
+impl ItemVariantQueries {
+    pub async fn item_variants_configured(
+        &self,
+        ctx: &Context<'_>,
+        store_id: String,
+    ) -> Result<bool> {
+        let user = validate_auth(
+            ctx,
+            &ResourceAccessRequest {
+                resource: Resource::QueryItems,
+                store_id: Some(store_id.clone()),
+            },
+        )?;
+
+        let service_provider = ctx.service_provider();
+        let service_context = service_provider.context(store_id.clone(), user.user_id)?;
+
+        let item_variants = service_provider
+            .item_service
+            .get_item_variants(&service_context, None, None, None)
+            .map_err(StandardGraphqlError::from_list_error)?;
+
+        Ok(item_variants.count > 0)
+    }
+}
+
```

Comment:
Is this simpler than adding an `itemVariant` filter to the `items` node and just getting totalCount? That would seem cleaner that adding a whole top-level node just for this, but if this is less code, then makes sense.